### ISSUE TITLE
Incorrect Excel parsing when system decimal separator is a comma

### ISSUE THREAD

**Issue by tmp64** [cathei/BakingSheet#38]
If the system is configured to use decimal separator different from a dot (`.`), Excel converter parses floats incorrectly.

Source Excel:  
![image](https://github.com/cathei/BakingSheet/assets/20199236/c3e90b8f-2d02-4e0c-b9be-495be26928fb)

Converted JSON:  
![image](https://github.com/cathei/BakingSheet/assets/20199236/b352f7c3-5562-4850-827a-604a857aa7f8)

I narrowed this down to a `ToString` call in `ExcelSheetConverter.Page.GetCell`:
```
Debugger console:
  _table.Rows[row][col]
  0.29999999999999999

  _table.Rows[row][col].ToString()
  "0,3"
```

I fixed it like this. Don't know if that's the only place to fix though.
```cs
public string GetCell(int col, int row)
{
	if (col >= _table.Columns.Count || row >= _table.Rows.Count)
		return null;
	
	if (_table.Rows[row][col] is IConvertible conv)
		return conv.ToString(CultureInfo.InvariantCulture);
	else
		return _table.Rows[row][col].ToString();
}
```

My regional settings for reference:  
![image](https://github.com/cathei/BakingSheet/assets/20199236/d00168cf-a053-415c-a279-bbe561f5e307)

**Comment by cathei** [comment]
Thank you for reporting! This should've been accounted with https://github.com/cathei/BakingSheet/issues/12, but I overlooked it.

Fixed in [v4.1.3](https://github.com/cathei/BakingSheet/releases/tag/v4.1.3)

**Comment by tmp64** [comment]
Thanks for fixing and releasing so quickly!
